### PR TITLE
Fix mwc-list-item

### PR DESCRIPTION
**Pull request by mathieuleclaire**

### CODE REVIEW

**Comment by mathieuleclaire** [comment]
The "Fix object apply function" should have been in a separate Pull Request; my bad.

File: generator/builder.sc [hunk 1]
```diff
@@ -241,7 +241,7 @@ class Builder(comp: WebComponent) {
 
         $buildStyles
 
-        def apply(mods: ModFunction*): HtmlElement = {
+        def apply(mods: ModFunction*): El = {
```
Comment (uosis) [inline generator/builder.sc:244]: if you make this a separate PR, we can merge it

File: generator/material.sc [hunk 2]
```diff
@@ -1307,7 +1307,7 @@ val componentCollection = WebComponentCollection(
           tag = "mwc-list-item",
           description = "List item",
           link = "https://github.com/material-components/material-components-web-components/tree/master/packages/list",
-          npmPackage = NpmPackage(name = "@material/mwc-list", version = "0.18.0"),
+          npmPackage = NpmPackage(name = "@material/mwc-list/mwc-list-item", version = "0.18.0"),
```
Comment (uosis) [inline generator/material.sc:1310]: Like I mentioned, this is not sufficient - this will result in correct import, but incorrect npm package. It works because you did not regenerate build.sbt.

**Review by uosis — CHANGES_REQUESTED** [review]

**Comment by uosis** [comment]
Fixed by 24b7179246ecee79a53d93edd857e11f61aabbee
